### DISCUSSION TITLE
Returning `plot_model` data

### DISCUSSION THREAD

**Discussion by HISSAOUI** [pycaret/pycaret#3522] [Q&A]
please, can I retrieve values in plot_model()

**Comment by ngupta23** [comment]
I think right now, this is only supported by time series - https://pycaret.readthedocs.io/en/stable/api/time_series.html#pycaret.time_series.plot_model

Use `return_data` = True

**Reply by HISSAOUI** [reply]
I am working on a classification problem. In the class pycaret.classification.ClassificationExperiment, the plot_model function does not contain the return_data argument

**Reply by ngupta23** [reply]
You can open an enhancement request in the Issues Section for the same.
